src/meshes/p4est_mesh.jl

```diff
@@ -299,6 +303,57 @@ function P4estMesh{NDIMS}(meshfile::String;
 end
 
 
+"""
+    P4estMesh(trees_per_face_dimension, layers, inner_radius, thickness;
+              polydeg, RealT=Float64,
+              initial_refinement_level=0, unsaved_changes=true)
+
+Build a "Cubed Sphere" mesh as `P4estMesh` with
+`6 * trees_per_face_dimension^2 * layers` trees.
+
+The mesh will have two boundaries, `:inside` and `:outside`.
+
+# Arguments
+- `trees_per_face_dimension::Integer`: the number of trees in the first two local dimensions of
+                                       each face.
+- `layers::Integer`: the number of trees in the third local dimension of each face, i.e., the number
+                     of layers of the sphere.
+- `inner_radius::Integer`: the inner radius of the sphere.
+- `thickness::Integer`: the thickness of the sphere. The outer radius will be `inner_radius + thickness`.
+- `polydeg::Integer`: polynomial degree used to store the geometry of the mesh.
+                      The mapping will be approximated by an interpolation polynomial
+                      of the specified degree for each tree.
+- `RealT::Type`: the type that should be used for coordinates.
+- `initial_refinement_level::Integer`: refine the mesh uniformly to this level before the simulation starts.
+- `unsaved_changes::Bool`: if set to `true`, the mesh will be saved to a mesh file.
```

Comment:
```suggestion
- `trees_per_face_dimension::Integer`: The number of trees in the first two local dimensions of
                                       each face.
- `layers::Integer`: The number of trees in the third local dimension of each face, i.e., the number
                     of layers of the sphere.
- `inner_radius::Integer`: The inner radius of the sphere.
- `thickness::Integer`: The thickness of the sphere. The outer radius will be `inner_radius + thickness`.
- `polydeg::Integer`: Polynomial degree used to store the geometry of the mesh.
                      The mapping will be approximated by an interpolation polynomial
                      of the specified degree for each tree.
- `RealT::Type`: The type that should be used for coordinates.
- `initial_refinement_level::Integer`: Refine the mesh uniformly to this level before the simulation starts.
- `unsaved_changes::Bool`: If set to `true`, the mesh will be saved to a mesh file.
```

Reply:
I've always done it like this because that's how it's done in the [Julia manual](https://docs.julialang.org/en/v1/manual/documentation/). Should I change it everywhere?

Reply:
No, it's ok. If you want to leave it like this, it's fine by me. Maybe my suggestions is not really helpful.